$OpenBSD: patch-Makefile,v 1.2 2019/03/13 01:14:47 jturner Exp $

Use a date stamp for JANET_BUILD to avoid using git.
Replace hardcoded -O2.
Fix installation.

Index: Makefile
--- Makefile.orig
+++ Makefile
@@ -26,15 +26,15 @@ PREFIX?=/usr/local
 
 INCLUDEDIR=$(PREFIX)/include
 BINDIR=$(PREFIX)/bin
-JANET_BUILD?="\"$(shell git log --pretty=format:'%h' -n 1)\""
+JANET_BUILD?="\"$(shell date +%Y%m%d)\""
 CLIBS=-lm
 JANET_TARGET=build/janet
 JANET_LIBRARY=build/libjanet.so
 JANET_PATH?=$(PREFIX)/lib/janet
-MANPATH?=$(PREFIX)/share/man/man1/
+MANPATH?=$(PREFIX)/man/man1/
 DEBUGGER=gdb
 
-CFLAGS=-std=c99 -Wall -Wextra -Isrc/include -fpic -O2 -fvisibility=hidden \
+CFLAGS=-std=c99 -Wall -Wextra -Isrc/include -fpic $(FLAGS) -fvisibility=hidden \
 	   -DJANET_BUILD=$(JANET_BUILD)
 LDFLAGS=-rdynamic
 
